src/Authenticator.js

```diff
@@ -122,8 +122,7 @@ class Authenticator {
       else if (!response.ok) throw new Error();
       else return response.json().then(auth => this._onSuccess(auth));
     }).catch(error => {
-      this._onError(error);
```

Comment:
When offline this causes the player to restart. As long as the API is up and you are connected you should get a 401